concrete/src/Updater/Migrations/Migrations/Version20211023155414.php

```diff
@@ -98,23 +98,24 @@ protected function upgradeImageValue(array $legacyValueValueData)
     protected function upgradeTypeValue(string $variable, array $legacyValueValueData)
     {
         $value = new TypeValue();
-        $colorStyle = new ColorStyle();
-        $colorStyle->setVariable($variable . '-type');
-        $colorValue = $this->upgradeColorValue($legacyValueValueData['color']);
-        $sizeStyle = new SizeStyle();
-        $sizeStyle->setVariable($variable . '-type');
-        $sizeValue = $this->upgradeSizeValue($legacyValueValueData['fontSize']);
-
-        $value->addSubStyleValue(new StyleValue($colorStyle, $colorValue));
-        $value->addSubStyleValue(new StyleValue($sizeStyle, $sizeValue));
-
+        if (is_array($legacyValueValueData['fontSize'])) {
```

Comment:
This is wrong, it's checking size but updating color.

Reply:
true. thanks for double checking.